pycaret/anomaly/functional.py

```diff
@@ -95,6 +97,14 @@ def setup(
         names.
 
 
+    index: bool, int, str or sequence, default=False
```

Comment:
can we add a one sentence description of the argument and what it does? It's not clear to me from the bullet points. Something like "How to handle index in the `data` dataframe"